packages/postgres-database/src/simcore_postgres_database/models/projects.py

```diff
@@ -141,6 +141,12 @@ class ProjectType(enum.Enum):
         default=False,
         doc="If true, the project is by default not listed in the API",
     ),
+    sa.Column(
+        "trashed_at",
+        sa.DateTime(timezone=True),
+        nullable=True,
+        doc="Timestamp indicating when the project was marked as trashed, or null otherwise.",
```

Comment:
minor: note that you doc this as "when" the project was marked as trashed, not with "at" ;)

Reply:
AFAIK `_at` is used as a prefix in many APIs to denote a snapshot in time (same as `_by` to denote some sort of user).   

I find it also convenient because it is very short. Nonetheless, if it is not clear, i am fine using `_when` and `_who`  suffixes.  The important thing is that we consistently use one standard across all the database. 

Asked for chatty:
>>While _when is less conventional than _at, it could still be used effectively if it fits the domain or makes sense to describe certain timestamps in your application. However, it might be less immediately recognizable to developers accustomed to the more standard _at convention